src/test/scala/redis/RedisSpec.scala

```diff
@@ -12,7 +12,7 @@ import org.specs2.concurrent.FutureAwait
 import org.specs2.mutable.SpecificationLike
 import org.specs2.specification.core.Fragments
 
-import scala.collection.JavaConverters._
+// import scala.collection.JavaConverters._
```

Comment:
This is renamed in Scala 2.13 and the new import is not backward compatible but fortunately the code that used it was also commented out.